### ISSUE TITLE
Sanity check with VALL-F after 17 epochs

### ISSUE THREAD

**Issue by primepake** [lifeiteng/vall-e#67]
After 17 epochs with VALL-F model the result looks like real (the result from unseen speakers).

1. promt: https://drive.google.com/file/d/1k7jy7LNV5HaBPeB_A8nwIbJOQJC-7oi9/view?usp=sharing
result: https://drive.google.com/file/d/1I6fnlEbt9pDEbkgElZXWP1b6gyOYFb8x/view?usp=sharing

2. promt: https://drive.google.com/file/d/1verQvU1SaS_i7bqN3nah1GsnMiibExQ7/view?usp=sharing
result: https://drive.google.com/file/d/1WCt8_2wvL7PVZHE4MulXmgt_JMM_pJCm/view?usp=sharing

3. promt: https://drive.google.com/file/d/12GYZhgMwPhQDdWBWYi_eakwVh4OS0E7W/view?usp=sharing
result: https://drive.google.com/file/d/1L7GN3xPKHA3TeQVEjB3yeTdZHxMVbq1s/view?usp=sharing

4. promt: https://drive.google.com/file/d/1U0KovG5g7wI-p2sExawCJm_EPAAW2HA-/view?usp=sharing
result: https://drive.google.com/file/d/1Is7Lumj2DiN0WmBQu3xp4PYlZBhIif8I/view?usp=sharing

**Comment by mingqilearning** [comment]
> After 17 epochs with VALL-F model the result looks like real (the result from unseen speakers).
> 
> 1. promt: https://drive.google.com/file/d/1k7jy7LNV5HaBPeB_A8nwIbJOQJC-7oi9/view?usp=sharing
>    result: https://drive.google.com/file/d/1I6fnlEbt9pDEbkgElZXWP1b6gyOYFb8x/view?usp=sharing
> 2. promt: https://drive.google.com/file/d/1verQvU1SaS_i7bqN3nah1GsnMiibExQ7/view?usp=sharing
>    result: https://drive.google.com/file/d/1WCt8_2wvL7PVZHE4MulXmgt_JMM_pJCm/view?usp=sharing
> 3. promt: https://drive.google.com/file/d/12GYZhgMwPhQDdWBWYi_eakwVh4OS0E7W/view?usp=sharing
>    result: https://drive.google.com/file/d/1L7GN3xPKHA3TeQVEjB3yeTdZHxMVbq1s/view?usp=sharing
> 4. promt: https://drive.google.com/file/d/1U0KovG5g7wI-p2sExawCJm_EPAAW2HA-/view?usp=sharing
>    result: https://drive.google.com/file/d/1Is7Lumj2DiN0WmBQu3xp4PYlZBhIif8I/view?usp=sharing

@primepake Hello, i have some questions. 
The model is train on libritts data?
would you share your train hyperparameter? For example,lr, decay, warmup, max-duration, etc. Thanks.

**Comment by primepake** [comment]
I just basically run the cmd below
```
python3 bin/trainer.py --manifest-dir ./data/tokenized \
--text-tokens ./data/tokenized/unique_text_tokens.k2symbols \
--max-duration 40 --filter-max-duration 40 --dtype "float32" \
--model-name "vallf" --norm-first true --add-prenet false \
--decoder-dim 1024 --nhead 16 --num-decoder-layers 12 --prefix-mode 0 \
--accumulate-grad-steps 1 --base-lr 0.05 \
--num-epochs 200 --start-epoch 1 --start-batch 0 \
--exp-dir exp/vallf/debug
```

**Comment by debasishaimonk** [comment]
@primepake I guess u have a very big gpu, as u had trained it on a filer max-duration of 40secs.
